src/seedu/addressbook/AddressBook.java

```diff
@@ -308,11 +308,20 @@ private static void setupDefaultFileForStorage() {
      * Returns true if the given file is acceptable.
      * The file path is acceptable if it ends in '.txt'
      * TODO: Implement a more rigorous validity checking.
+     * Note: Access modifier was changed from private to 
```

Comment:
The original method checks if the file path is acceptable, not if the file already exists. Does the current method follow the same?